packages/uicore/src/components/ListHeader/ListHeader.tsx

```diff
@@ -12,20 +12,24 @@ import css from './ListHeader.css'
 import { Layout, Text } from '../../'
 import { SortDropdown, SortDropdownProps } from '../SortDropdown/SortDropdown'
 
-interface ListHeaderProps extends SortDropdownProps {
+export interface ListHeaderProps extends SortDropdownProps {
   totalCount?: number
   className?: string
+  content?: React.ReactNode
 }
 
 export function ListHeader(props: ListHeaderProps): JSX.Element {
-  const { totalCount, className, ...rest } = props
+  const { totalCount, className, content, ...rest } = props
   return (
     <Layout.Horizontal
       flex={{ distribution: 'space-between' }}
       margin={{ bottom: 'small' }}
       className={cx(css.listHeader, className)}>
       <Text font={{ variation: FontVariation.H5 }}>Total: {totalCount}</Text>
-      <SortDropdown {...rest} />
+      <Layout.Horizontal>
```

Comment:
can we add spacing small or medium for horizontal layout here?

Reply:
@shivanand-harness it will be the responsibility of content how to add spacing. It is same as PageHeader.
https://github.com/harness/uicore/blob/main/packages/uicore/src/components/Page/PageHeader.tsx#L56